tests/test_distributedarray.py

```diff
@@ -120,6 +121,28 @@ def test_to_dist(par):
     assert dist_array.axis == par['axis']
 
 
+@pytest.mark.mpi(min_size=2)
+@pytest.mark.parametrize("par", [(par1), (par1j), (par2),
+                                 (par2j), (par3), (par3j)])
+def test_local_shapes(par):
+    """Test the `local_shapes` parameter in DistributedArray"""
+    loc_shapes = MPI.COMM_WORLD.allgather(local_split(par['global_shape'],
+                                                      MPI.COMM_WORLD, par['partition'], par['axis']))[::-1]
```

Comment:
same question as before, why `[::-1]`

Reply:
This was basically to check if our local_shapes feature is working fine(I don't think we can add raw numbers as num_processes will be different)..., I forgot to remove [::-1] as I was just testing..
so lets take an example after gathering local_split with global_shape = (10, 10) -> rank0 = (4, 10) -> rank1 = (3, 10) -> rank2 = (3, 10)
collecting all will give [(4, 10),  (3, 10), (3, 10)](which is default) I wanted to test for different local_shapes so I reversed the list giving [(3, 10), (3, 10), (4, 10)] which thus local_shape at each rank were switched and tried testing which worked fine...

Reply:
Great, I had assumed that was the case (which is a good test to do) but I just wanted to be sure… I would say either remove it or leave it as but just add a small comment

Reply:
Just added a small comment